portfolio/src/main/webapp/comments.css

```
@@ -53,24 +54,28 @@ input[type=submit] {
   margin: none;
   padding: 5px 5px;
   text-decoration: none;
+  text-align: center;
```

```suggestion
  text-align: center;
  text-decoration: none;
```
alphabetize here too